.github/workflows/android-perf.yml

```diff
@@ -0,0 +1,179 @@
+name: android-perf
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  pull_request:
+  push:
+    tags:
+      - ciflow/perf-android
+  # Note: GitHub has an upper limit of 10 inputs
+  workflow_dispatch:
+    inputs:
+      models:
+        description: Models to be benchmarked
+        required: false
+        type: string
+        default: stories110M
+      devices:
+        description: Target devices to run benchmark
+        required: false
+        type: string
+        default: false
+      delegates:
+        description: Backend delegates
+        required: false
+        type: string
+        default: xnnpack
+      threadpool:
+        description: Run with threadpool?
+        required: false
+        type: boolean
+        default: false
+      benchmark_configs:
+        description: The list of configs used the benchmark
+        required: false
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}-${{ github.event_name == 'schedule' }}
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  export-models:
+    name: export-models
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    strategy:
```

Comment:
Nit: Do you plan to run the job with more than one model here?  it doesn't look that way from what I see here, so I think we could get rid of matrix strategy for simplicity (because there is only one model).  You can use `${{ github.event.inputs.models }}` directly in the script as the model name

Reply:
Ah, never mind, I guess you want to extend this in the future with other inputs too, i.e. `devices`, `delegates` as they are not used atm.  So, `strategy` is ok

Reply:
Yes, I will want to be able to support passing multiple model names and run benchmark for each model. That way we may be using the same workflow for both on-demand benchmark (typically one model) and periodic benchmarking (all models).